.babelrc

```diff
@@ -1,13 +1,4 @@
 {
-  "presets": [
-    [
-      "@babel/preset-env",
-      {
-        "targets": {
-          "node": "current"
-        }
-      }
-    ]
-  ],
-  "plugins": ["@babel/plugin-proposal-class-properties"]
```

Comment:
This is deprecated since its functionality was included into ECMAScript standard.